frontend/src/metabase/visualizations/lib/settings.ts

```diff
@@ -31,10 +32,113 @@ const WIDGETS = {
   colors: ChartSettingColorsPicker,
 };
 
+// VisualizationSettings?
+type ComputedSettings = {
+  currency?: string;
+  date_style?: string;
+  date_separator?: string;
+  date_abbreviate?: boolean;
+  currency_in_header?: boolean;
+  currency_style?: string;
+  number_style?: string;
+  time_enabled?: boolean;
+  [key: string]: any;
+};
+
+// VisualizationSettingDefinition?
+export type SettingDef = {
```

Comment:
There is a type for this already, can you please tweak it if it is wrong? 
`VisualizationSettingDefinition` in `frontend/src/metabase/visualizations/types/visualization.ts`
It is stored in visualizations because each visualization exposes settings definitions by a static `settings` variable

Reply:
Yes, thank you!  I had it in the comment above as reference, since I realized it too late, will converge on putting them together.